relbench/datasets/event.py

```diff
@@ -23,9 +23,9 @@ class EventDataset(RelBenchDataset):
     )
 
     train_start_timestamp = pd.Timestamp("2012-06-20")
-    val_timestamp = pd.Timestamp("2012-11-14")
-    test_timestamp = pd.Timestamp("2012-11-28")
-    max_eval_time_frames = 2
+    val_timestamp = pd.Timestamp("2012-11-21")
+    test_timestamp = pd.Timestamp("2012-11-29")
```

Comment:
nit: why don't we make timedelta 7 days here?

Reply:
If I make it 7 days, I get the following error 
```
RuntimeError: val timestamp + timedelta is larger than max timestamp! This would cause val labels to be generated with insufficient aggregation time.
```

Reply:
Looking more into the code,
```
(Pdb) self.dataset.db.max_timestamp
Timestamp('2012-11-27 23:55:55.935000')
(Pdb) self.dataset.val_timestamp
Timestamp('2012-11-21 00:00:00')
(Pdb) self.dataset.val_timestamp + self.timedelta
Timestamp('2012-11-28 00:00:00')
```
I am trying to avoid changing the relbench data processing code, so I just added 1 day in the test_timestamp here.

Reply:
Ok, thanks for clarifying